docs/user-guide/settings/advanced.md

```diff
@@ -173,4 +242,13 @@ In addition, PhotoPrism also supports TIFF, PNG, BMP and GIF files. Be aware tha
 
 !!! info
     Generated sidecar files will be stored outside your originals folder by default, so that
-    RAW to JPEG conversion also works in read-only mode.
\ No newline at end of file
+    RAW to JPEG conversion also works in read-only mode.
+
+### JPEG Size Limit
+
+This controls the maximum size of downsampled JPEG preview files to create
+when converting original RAW images.
+
+### Use Presets
+
+Disables simultaneous conversion of RAW files to apply Darktable presets.
```

Comment:
I wasn't sure what this really means.

I think:

> PhotoPrism uses [Darktable](https://www.darktable.org/) behind the scenes to convert RAW images to JPEG. 

But I don't understand what the "preset values" are or means. What happens if I don't tick this box? What would be used instead of "presets"?

Reply:
Defaults

Reply:
Okay, thanks. Are the defaults configurable or viewable anywhere? And where can I set the presets to override the defaults (if at all)?

Reply:
Good question, it's actually more a question about Darktable than PhotoPrism. A contributor told us to add this option so that converting RAW files is faster.

Reply:
Testing how exactly this affects image style & quality is on our todo.